docs/network-flow-visibility.md

```diff
@@ -0,0 +1,101 @@
+| destinationClusterIP      | 55829         | 106      | ipv4Address |
+| destinationServicePort    | 55829         | 107      | unsigned16  |
+| destinationServicePortName| 55829         | 108      | string      |
+
+[Elasticsearch](https://www.elastic.co/elasticsearch/), as a RESTful search
+engine, supports storing, searching and indexing records received. 
+[Kibana](https://www.elastic.co/kibana/) is mainly for data visualization and
+exploration.
+
+## Deployment Steps
+To create all the necessary resources in the `elk-flow-collector` namespace
+and get everything up-and-running, run:
+```shell
+kubectl create namespace elk-flow-collector
+kubectl create configmap logstash-configmap -n elk-flow-collector --from-file=build/yamls/elk-flow-collector/logstash/
+kubectl apply -f build/yamls/elk-flow-collector/elk-flow-collector.yml -n elk-flow-collector
+```
+Kibana dashboard is exposed as a Nodeport Service, which can be accessed via
+`http://[NodeIP]: 30007`
+
+`build/yamls/flow/kibana.ndjson` is an auto-generated reusable file containing 
+pre-built objects for visualizing pod-to-pod, pod-to-service and node-to-node 
+flow records. To import the dashboards into Kibana, go to 
+**Management -> Saved Objects** and import `build/yamls/flow/kibana.ndjson`.
+
+
+## Pre-built Dashboards
+The following dashboards are pre-built and recommended for Antrea flow
+visualization.
+
+### Overview
+An overview of pod-based flow records information is provided.
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/flow-visualization-overview.png" width="900" alt="Flow
+Visualization Overview Dashboard"> 
+
+### Flows
+#### Pod-to-pod Traffic
+Pod-to-pod Tx and Rx traffic is shown in sankey diagrams. Corresponding 
+source/destinationpod throughput is visualized using stacked line graph. 
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/flow-visualization-flow-1.png" width="900" alt="Flow
+Visualization Flows Dashboard"> 
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/flow-visualization-flow-2.png" width="900" alt="Flow
+Visualization Flow Dashboard"> 
+
+#### Pod-to-service Traffic
+Pod-to-service traffic is presented similar to pod-to-pod traffic.
+Corresponding source/destination IP addresses are shown in tooltips.
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/flow-visualization-flow-3.png" width="900" alt="Flow
+Visualization Flows Dashboard"> 
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/flow-visualization-flow-4.png" width="900" alt="Flow
+Visualization Flow Dashboard"> 
+
+### Flow Records 
+Flow Records dashboard shows raw flow records over time with filter.
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/flow-visualization-flow-record.png" width="900" alt="Flow
+Visualization Flow Record Dashboard">
+
+### Node Throughput
+Node Throughput dashboard visualizes inter-node and intra-node traffic
```

Comment:
... shows the visualization of inter-node and intra-node traffic by aggregating all the pod traffic per each node.